packages/react-shortcuts/src/Shortcut/hooks.ts

```diff
@@ -11,23 +11,22 @@ export interface Options {
   held?: HeldKey;
   node?: HTMLElement | null;
   ignoreInput?: boolean;
+  ignoreTags?: HTMLElement['tagName'][];
   allowDefault?: boolean;
 }
 
+const DEFAULT_IGNORE_TAGS = ['INPUT', 'SELECT', 'TEXTAREA'];
+
 export default function useShortcut(
   ordered: Key[],
   onMatch: (matched: {ordered: Key[]; held?: HeldKey}) => void,
   options: Options = {},
 ) {
   const shortcutManager = React.useContext(ShortcutContext);
   const subscription = React.useRef<Subscription | null>(null);
-  const {node, held, ignoreInput, allowDefault} = options;
+  const {node, held, ignoreInput, ignoreTags, allowDefault} = options;
 
   React.useEffect(() => {
-    if (node != null) {
```

Comment:
I believe the `node` feature was not working all together as this early return was preventing the shortcut to be registered when a `node` was provided 🤔